scripts/audit-deps.js

```diff
@@ -61,14 +61,16 @@ if (auditReport) {
     }
   }
 
-  for (const advId of Object.keys(auditReport.advisories)) {
-    const adv = auditReport.advisories[advId];
-
-    if (exceptions.includes(adv.url)) {
-      ignoredIssues.push(adv);
-      continue;
+  if (auditReport.advisories) {
+    for (const advId of Object.keys(auditReport.advisories)) {
+      const adv = auditReport.advisories[advId];
+
+      if (exceptions.includes(adv.url)) {
+        ignoredIssues.push(adv);
+        continue;
+      }
+      blockingIssues.push(adv);
     }
-    blockingIssues.push(adv);
   }
```

Comment:
I think this part may be because of a change in the json format returned by npm audit in more recent npm versions, I have noticed recently while I was investigating some other bug.

The actual fix would be to actually collect the advisories from the new json format, let's keep this out of this PR, I'll file an issue and deal with it in a separate PR.

Reply:
It was causing CI to fail so added a fix but I agree it should be landed in a separate PR